### ISSUE TITLE
Provide a way to dump the TF schema of a bridged provider

### ISSUE THREAD

**Issue by VenelinMartinov** [pulumi/pulumi-terraform-bridge#1982]
## Hello!

- Vote on this issue by adding a 👍 reaction
- If you want to implement this feature, comment to let us know (we'll work with you on design, scheduling, etc.)

## Issue details

It'd be very useful if a provider binary had a flag to dump the TF schema. This would be invaluable for bug reports against dynamically bridged providers and could also be useful for using the TF schema for analytics purposes.

**Comment by VenelinMartinov** [comment]
Should the provider binary be responsible for that or should the binary dump some JSON blob which we can then post-process into whatever form we want?

**Comment by t0yv0** [comment]
There's MarshallableProviderInfo that has mapping information including, I think, TF schema, but it's worth double checking if it is missing anything interesting for us. 

You can get it with a flag already:

```
	dumpInfo := flags.Bool("get-provider-info", false, "dump provider info as JSON to stdout")
```

For quick and dirty,  TF schema itself has a gRPC representation (and every gRPC has associated JSON form). That could be used. 

Decisions to be made how to tabularize this though can indeed affect usability for our query scenarios, but we can draw on Pablo's expertise possibly.

**Comment by VenelinMartinov** [comment]
Taking it back: the provider info does not contain information about property constraints like `ExactlyOneOf` and `ConflictsWith` which would be useful for https://github.com/pulumi/pulumi-terraform-bridge/issues/1949

**Comment by t0yv0** [comment]
Did we land this? Curious.

**Comment by VenelinMartinov** [comment]
No, https://github.com/pulumi/pulumi-terraform-bridge/pull/2315 is still open

**Comment by VenelinMartinov** [comment]
We do have some dashboards based on the draft though
